src/integrations/lofn/clients.ts

```diff
@@ -0,0 +1,33 @@
+import {
+  ILofnClient,
+  LofnFeedConfigPayload,
+  LofnFeedConfigResponse,
+} from './types';
+import { RequestInit } from 'node-fetch';
+import { fetchOptions as globalFetchOptions } from '../../http';
+import { retryFetchParse } from '../retry';
+
+export class LofnClient implements ILofnClient {
+  private readonly url: string;
+  private readonly fetchOptions: RequestInit;
+
+  constructor(
+    url = process.env.LOFN_ORIGIN,
+    fetchOptions: RequestInit = globalFetchOptions,
+  ) {
+    this.url = url;
+    this.fetchOptions = fetchOptions;
+  }
+
+  fetchConfig(payload: LofnFeedConfigPayload): Promise<LofnFeedConfigResponse> {
+    return retryFetchParse(
```

Comment:
Optional, since it's new maybe we'd like specific logging for fails?

Reply:
Added in 765337e984221288565f15363bbde8457940b2c7